### ISSUE TITLE
Interactive demo based on streamlit

### ISSUE THREAD

**Issue by mik-laj** [tobymao/sqlglot#927]
Hello.

I am very impressed with this project and I am glad that it is being developed. At first, I was skeptical about it and before I used it for the first time it took a while. I was thinking about what would convince me and I think a web version would be a big booster for me.

Have you thought about preparing one? I think it shouldn't be difficult using [Streamlit framework](https://docs.streamlit.io/library/get-started). About 20 lines.
```python
import sqlglot
import streamlit as st
from streamlit_ace import st_ace

all_dialects = sorted(d.value for d in iter(sqlglot.Dialects) if d.value)

left, right = st.columns(2)
input_lng = left.selectbox("Input language",  all_dialects)
output_lng = right.selectbox("Output language",  all_dialects)
pretty = st.checkbox("Pretty format")
input_sqls = st_ace(language="sql")

if input_sqls:
    try:
        output_sqls = sqlglot.transpile(input_sqls, read=input_lng, write=output_lng, pretty=pretty)
        for output_sql in output_sqls:
            st.code(output_sql, language="sql")
    except sqlglot.errors.ParseError as e:
        st.write("**Transpilation errors:**")
        st.table(e.errors)

```
<img src="https://user-images.githubusercontent.com/12058428/212211130-8188ec38-517c-410e-963c-5c5439e05ac2.png"  width="400">
Plus, you can deploy it for free on Streamlit Cloud and automatic deployment from Github will ensure that maintenance is cost-free.

**Comment by tobymao** [comment]
this is cool but out of the scope of this project

**Comment by mik-laj** [comment]
@tobymao I agree that building a complex web application that provides all the features of this library is out of scope. But here I proposed to prepare a simple demo that shows how this library behaves. It's more of a promotional material for the library than a tool that will be used in the real world. Thanks to this, the user can click links and see how this library works without the need to install additional things on the computer.

Similarly, other projects have playgrounds that show how their libraries work.
* Typescript: https://www.typescriptlang.org/
* ESLint: https://eslint.org/play/
* Black: https://black.vercel.app/

I have deployed the above-pasted code from my account to my account, but it won't be updated probably. 
https://mik-laj-sqlglot-demo-streamlit-app-vegau4.streamlit.app/
I think it would be better if the project maintainers had control over it and could update it with the release of a new version. In particular, such a link could be added to README.md.
